### DISCUSSION TITLE
Why user organizations list is empty?

### DISCUSSION THREAD

**Discussion by pranavirabatti** [Sunbird-Lern/Community#11] [Issues]
Site Environments Details - SunbirdEd portal of version 4.4.1

I've created a new user using the signup process. And we configured the default organization also. Now when I checked the user details I found the organization's list is empty.

![image](https://user-images.githubusercontent.com/22250006/172554259-63100b6a-0bf0-4bd2-8743-a0a2908ee992.png)

@amit-tarento please check this. Anything we left to configure?

cc: @Renuka-S, @komalm

**Comment by pranavirabatti** [comment]
Any update on this?

**Reply by amitec006** [reply]
Can you provide me more details like which release version and the curl request which you used to create user.

**Reply by pranavirabatti** [reply]
I have mentioned the sunbird version at the top. I've created this user using the sign-up process. 

```
curl 'https://domain_name/learner/user/v2/signup' \
  -H 'Accept: application/json' \
  -H 'Accept-Language: en-GB,en;q=0.9' \
  -H 'Connection: keep-alive' \
  -H 'Content-Type: application/json' \
  -H 'Cookie: connect.sid=s%3AxrFTACCEz7dPprlq41uSzMq0r24RpJDa.0DtBG0ZlYpDN%2FuqgU7vNS%2FwGsYxPmTpHhJVmvlGjYSs' \
  -H 'Origin: https://domain_name' \
  -H 'Referer: https://domain_name/signup?client_id=portal&state=cabde6c4-b464-4a72-b2b5-5525233a2ee6&redirect_uri=https:%2F%2Fdomain_name%2Fresources%3Fauth_callback%3D1&scope=openid&response_type=code&version=4&error_callback=https:%2F%2Fdomain_name%2Fauth%2Frealms%2Fsunbird%2Fprotocol%2Fopenid-connect%2Fauth' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Site: same-origin' \
  -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36' \
  -H 'X-App-Id: dev.dev.portal' \
  -H 'X-App-Version: 4.4.1' \
  -H 'X-Channel-Id: 0134866036584857600' \
  -H 'X-Device-ID: c4e7ab0c5351aac3741b699ac3ef9cfd' \
  -H 'X-Org-code: 0134866036584857600' \
  -H 'X-Request-ID: 5cd6c107-07a2-98b5-b23f-435e2fd0d4f7' \
  -H 'X-Session-ID: dd197ca0-b7f0-5f9f-7e03-c6a49fe90b10' \
  -H 'X-Source: web' \
  -H 'X-msgid: 5cd6c107-07a2-98b5-b23f-435e2fd0d4f7' \
  -H 'sec-ch-ua: " Not A;Brand";v="99", "Chromium";v="100", "Google Chrome";v="100"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "Linux"' \
  -H 'ts: 2022-06-09T11:41:32+05:30' \
  --data-raw '{"params":{"source":"portal","signupType":"self"},"request":{"firstName":"TestJohnUser","password":"randompassword,"dob":"1989","email":"dehag50985@nzaif.com","emailVerified":true,"reqData":"{\"iv\":\"9a0f7a8e4f987f9e10ec1576ec7d3f3e\",\"encryptedData\":\"51a2860a1580417676ae801c6971612bacc4ccd243554d8c4431eb015bba1b8c02572547f910fb55d1a012b7c56ddd2a06c79692181ebe3b61e7a2bcf4271114\"}"}}' \
  --compressed
```

**Reply by amitec006** [reply]
Can you please check for error logs you are getting while making this api call.

**Reply by pranavirabatti** [reply]
I think you did not get my point. I'm not getting any errors. I'm able to create new users using self sign up process. Users which are created have no organization added to the list which I've shown you above though I can see the default root organization is mapped right at the bottom of the response. Ideally, there should be a default custodian organization in the organization's list?

**Reply by amitec006** [reply]
yes i know , i asking for error log which might occurred during inserting data in user_organisation table. so please fire the request again and capture the all logs related to it.then only we can analyse.

**Reply by pranavirabatti** [reply]
Ohkay. I will check. Thanks!

**Reply by pranavirabatti** [reply]
@amit-tarento I have skimmed through the logs. I did not find anything unusual whilst user signup process. Can you please check this? Recorded activities are of one user only.
[UserSignupLog.txt](https://github.com/Sunbird-Lern/Community/files/8887851/UserSignupLog.txt)

**Reply by amitec006** [reply]
@pranavirabatti There was an issue in release-4.4.1 of https://github.com/project-sunbird/sunbird-lms-service, instead of release-4.4.1 please use release-4.5.0.

**Reply by pranavirabatti** [reply]
We've upgraded it. It works now. Thanks @amit-tarento
